ocs/ocp.py

```diff
@@ -277,4 +277,9 @@ def exec_ceph_cmd(ceph_cmd):
     ocp_pod_obj = OCP(kind='pods', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
     ct_pod = get_ceph_tools_pod()
     ceph_cmd += " --format json-pretty"
-    return ocp_pod_obj.exec_cmd_on_pod(ct_pod, ceph_cmd).toDict()
+    out = ocp_pod_obj.exec_cmd_on_pod(ct_pod, ceph_cmd)
+
+    # For some commands, like "ceph fs ls", the returned output is a list
+    if isinstance(out, list):
+        return [item.toDict() for item in out if item]
+    return out.toDict()
```

Comment:
I'm not sure what we are trying to achieve here.

Moreover it's going to fail for simple outputs, eg. for command `ceph osd pool ls --format json` out is `["rbd"]` and then it fails on calling `toDict()` on the string object "rbd".

Reply:
@mbukatov we to toDict() here on the output of exec_cmd_on_pod(), that returns the output of exec_oc_cmd() which returns a Munch obj.
In case of the command you mentioned, this is exactly the reason for this patch. I was informed by @prsurve that in case of executing a ceph command that returns a list, toDict() will fail.

Reply:
LGTM 

Used for executing "ceph fs ls"

Result:-
Executing command: oc -n rook-ceph --kubeconfig /home/prsurve/OCS4/tmp/may28/auth/kubeconfig rsh rook-ceph-tools-5bd44d9bf8-nc576 ceph fs ls --format json-pretty
19:19:54 - MainThread - tests.ocs-288 - INFO - [{'name': 'ocsci-cephfs', 'metadata_pool': 'ocsci-cephfs-metadata', 'metadata_pool_id': 88, 'data_pool_ids': [89], 'data_pools': ['ocsci-cephfs-data0']}]